lib/go-tc/deliveryservice_requests.go

```diff
@@ -435,6 +435,58 @@ type DeliveryServiceRequestNullable struct {
 	XMLID           *string                     `json:"-" db:"xml_id"`
 }
 
+// Downgrade will convert an instance of DeliveryServiceRequestV41 to DeliveryServiceRequestV40
+func (dsr DeliveryServiceRequestV41) Downgrade() DeliveryServiceRequestV40 {
+	var dsrV40 DeliveryServiceRequestV40
+	dsrV40.Assignee = copyStringIfNotNil(dsr.Assignee)
+	dsrV40.AssigneeID = copyIntIfNotNil(dsr.AssigneeID)
+	dsrV40.Author = dsr.Author
+	dsrV40.AuthorID = copyIntIfNotNil(dsr.AuthorID)
+	dsrV40.ChangeType = dsr.ChangeType
+	dsrV40.CreatedAt = dsr.CreatedAt
+	dsrV40.ID = copyIntIfNotNil(dsr.ID)
+	dsrV40.LastEditedBy = dsr.LastEditedBy
+	dsrV40.LastEditedByID = copyIntIfNotNil(dsr.LastEditedByID)
+	dsrV40.LastUpdated = dsr.LastUpdated
+	if dsr.Original != nil {
+		dsrV40.Original = new(DeliveryServiceV40)
+		dsrV40.Original = &dsr.Original.DeliveryServiceV40
+	}
+	if dsr.Requested != nil {
+		dsrV40.Requested = new(DeliveryServiceV40)
+		dsrV40.Requested = &dsr.Requested.DeliveryServiceV40
+	}
+	dsrV40.Status = dsr.Status
+	dsrV40.XMLID = dsr.XMLID
+	return dsrV40
+}
+
+// Upgrade will convert an instance of DeliveryServiceRequestV40 to DeliveryServiceRequestV41
```

Comment:
GoDoc should end with punctuation. Also, it'd be helpful to note that the Requested and/or Original DSes are shallow-copied in this case.